.github/workflows/snyk-pr.yml

```diff
@@ -0,0 +1,31 @@
+name: snyk-pr
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  whitesource:
```

Comment:
Does this mean that we can get rid of the [other whitesource action](https://github.com/snowflakedb/snowflake-connector-python/blob/37443fb4cd0c548df0efe62199d2b52b324abe6e/.github/workflows/build_test.yml#L49)?

Reply:
yes, I have removed those files from the repo as well